depression_anxiety_stress_scales_21survey/survey.tsv

```diff
@@ -0,0 +1,24 @@
+instruction	Welcome to this survey. Press <strong>Next</strong> to begin.	0	1		
```

Comment:
This one is missing the top header and not loading correctly - I'll need to debug it.